src/App.jsx

```diff
@@ -1,40 +1,29 @@
-import React from 'react';
+import { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
according to task requirements you should save only `id`
![image](https://github.com/mate-academy/react_tabs-js/assets/86923089/af05b3d0-4b8a-4e32-9cca-9b1daf9cd04d)

then create a function to find a tab by `tab.id` to use here and inside `Tabs` component

explanation: in the task, the content of the tab is just a small string to simplify the task. But content might be another component etc. and it's not the best practice to save it in the state